Entwurf/Dokumentation.tex

```diff
@@ -0,0 +1,98 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage[german]{babel}
+\usepackage[T1]{fontenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\usepackage{graphicx}
+\usepackage[table,x11names]{xcolor}
+\newcommand\sbullet[1][.5]{\mathbin{\vcenter{\hbox{\scalebox{#1}{$\bullet$}}}}}
+\graphicspath{ {/home/ducio/Desktop/Se/exam-planner-documentation/Entwurf} }
+
+\begin{document}
+
+Datenbank für das Applikation besteht aus  vier Tabele : Group, Reservation, Appointment und Booking.
```

Comment:
Die Datenbank für die Applikation besteht aus vier Tabellen

Reply:
Bei der Einleitung solltest du direkt Bezug nehmen auf das Diagramm: Die Struktur der Tabellen ist in Abbildung \ref{fig:erm-diagram} dargestellt.